azure-mvn-settings.xml

```diff
@@ -1,8 +1,33 @@
 <settings>
+  <servers>
+    <server>
+      <id>azure-maven-feed</id>
+      <configuration>
+        <httpHeaders>
+          <property>
+            <name>Authorization</name>
+            <!-- Treat this auth token like a password. Do not share it with anyone, including Microsoft support. -->
+            <!-- The generated token expires on or before 28/07/2019 -->
+            <!-- This token should be pased here, then the resulting file uploaded as an azure secret file under the name secret-maven-settings.xml -->
+            <!-- This token should never be commited to a repo -->
+            <value>ENTER_TOKEN_HERE</value>
+          </property>
+        </httpHeaders>
+      </configuration>
+    </server>
+  </servers>
   <profiles>
     <profile>
       <id>google-mirror-jboss-proxy</id>
       <repositories>
+        <repository>
+          <id>azure-maven-feed</id>
+          <name>Quarkus Maven Feed</name>
+          <url>https://pkgs.dev.azure.com/quarkus-ci/_packaging/QuarkusMavenFeed/maven/v1</url>
```

Comment:
the token aspect is going to be fun as potentially local build and CI build will use different artifacts.

Also:

* if it a maven central mirror, if so what's the latency
* does it provide a mirror for "oss.snapshot"
* how does it work for publication?